organising/index.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Organising LRUG
+---
+
+# Organising LRUG
+
+LRUG doesn't happen spontaneously every month; it takes care and attention to detail to make sure it runs smoothly.  These pages are here to help you if it's your turn to organise one of our meetings, or if you're curious about how LRUG is run and are interested in helping out.
+
+## Guiding principles
+
+1. We document what we do, so everyone knows how to do everything
+2. We share the load, no one person on the team is expected to do everything
+3. We embrace randomness, we let the fates decide who owns a given task for a given meeting
+4. Ownership doesn't mean doing, if you own a task, you can delegate and ask for help
+
+## Tools
+
+We use lots of software to run LRUG:
+
+* [Slack][slack] - this is where all the organisers hang out; you can ask for help in here
+* [Harmonia][harmonia] - we use harmonia to randomly assign organisers to recurring tasks and adhoc events (like moderating email)
+* [Shared inbox][shared-inbox] - a shared email inbox where all the emails to organiser addresses go
+* [Schedule spreadsheet][schedule-spreadsheet] - this spreadsheet has a summary of all the meetings, the volunteer speakers list, a list of potential sponsors, and some email templates.
+* [Mailing list admin interface][mailing-list-admin] - this is where we deal with the mailing list member list and other settings
+* [Mailing list moderation interface][mailing-list-moderation] - this is where we deal with the mailing list moderation
+* [lrug.org github repo][github-lrug.org] - the repo for the main lrug website
+* [readme.lrug.org github repo][github-readme.lrug.org] - the repo for this site
+
+## Tasks
+
+Organising LRUG is split into two sets of tasks:
+
+1. [Organising a meeting](#organising-a-meeting)
+2. [Maintaining the community](#maintaining-the-community)
+
+### Organising a meeting
+
+> Note that this documents our current procedures during the COVID-19 pandemic where we're running remote meetings.  It'll be slightly different when we return to in-person meetings.
+
+Someone needs to:
```

Comment:
```suggestion
We need to:
```

I wonder if using "We" makes it a bit less likely that it will be read as "One single person needs to:"?